tribits/core/installation/TribitsPackageConfigTemplate.cmake.in

```diff
@@ -45,6 +45,15 @@
 #
 ##############################################################################
 
+if(CMAKE_VERSION VERSION_LESS 3.3)
+  set(${PDOLLAR}{CMAKE_FIND_PACKAGE_NAME}_NOT_FOUND_MESSAGE
+    "${PACKAGE_NAME} requires CMake 3.3 or later for 'if (... IN_LIST ...)'"
+    )
+  set(${PDOLLAR}{CMAKE_FIND_PACKAGE_NAME}_FOUND FALSE)
+  return()
+endif()
+cmake_minimum_required(VERSION 3.3...${CMAKE_VERSION})
```

Comment:
I would strongly suggest against setting an upper bound of `${CMAKE_VERSION}`. You're giving CMake a blank check to set any future policies to `NEW` even if you don't know what they do. I would suggest hard-coding the latest current version of CMake that you've tested against.

Reply:
Okay, I will change this to the minimum required TriBITS version:

```
cmake_minimum_required(VERSION 3.3...${TRIBITS_CMAKE_MINIMUM_REQUIRED})
```

That will eliminate maintenance issues since that gets updated whenever the min required version gets updated in:

https://github.com/TriBITSPub/TriBITS/blob/master/tribits/core/package_arch/TribitsConstants.cmake#L42

Reply:
Sounds good to me.